src/native/corehost/host_runtime_contract.h

```diff
@@ -51,6 +51,12 @@ struct host_runtime_contract
     const void* (HOST_CONTRACT_CALLTYPE* pinvoke_override)(
         const char* library_name,
         const char* entry_point_name);
+
+    // Probe the host for `path`. Sets pointer to data start and its size, if found.
+    bool(HOST_CONTRACT_CALLTYPE* external_assembly_probe)(
```

Comment:
@grendello I moved this to the end of the struct to keep the existing order. I don't think this should require updating your changes in dotnet/android, since they use designators.